packages/account-sdk/src/browser-entry.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Browser entry point for Base Account SDK
+ * This file exposes the account interface to the global window object
+ */
+
+import { createBaseAccountSDK } from './interface/builder/core/createBaseAccountSDK.js';
+import { base } from './interface/payment/base.js';
+import { CHAIN_IDS, TOKENS } from './interface/payment/constants.js';
+import { getPaymentStatus } from './interface/payment/getPaymentStatus.js';
+import { pay } from './interface/payment/pay.js';
+import type {
+  InfoRequest,
+  PayerInfo,
+  PaymentOptions,
+  PaymentResult,
+  PaymentStatus,
+  PaymentStatusOptions,
+} from './interface/payment/types.js';
+
+// Expose to global window object
+if (typeof window !== 'undefined') {
+  (window as any).base = base;
```

Comment:
it feels like `base` should be the top level window property here and everything is under it.

something like

window.base = {
   pay,
   getPaymentStatus,
   createBaseAccountSDK
}

Reply:
@montycheese Our thinking is to have 'base' be an end developer extraction with implementations that may be 'base chain only', may include product opinion, may not give full control etc.